.env.example

```
@@ -14,7 +14,8 @@ JWT_REFRESH_EXPIRES_IN=14d
 # AWS
 AWS_PUBLIC_BUCKET_NAME=
 AWS_ACCESS_KEY_ID=
-AWS_SECRET_ACCESS_KEY= 
+AWS_SECRET_ACCESS_KEY=
+AWS_REGION=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the AWS_SECRET_ACCESS_KEY key

(UnorderedKey)